src/main/java/dk/cngroup/lentils/repository/HintTakenRepository.java

```diff
@@ -22,6 +22,11 @@
             nativeQuery = true)
     List<HintTaken> findAllByTeamAndCypher(Long teamId, Long cypherId);
 
+    @Query(value = "SELECT ht FROM HintTaken ht " +
```

Comment:
toto funguje i bez `*` (`SELECT ht.*`)? Jestli jo tak to tak klidne nechej, jen jsem to doted nevedel ;)

Reply:
https://www.baeldung.com/spring-data-jpa-query
myslim ze hvezdicka musi byt jen kdyz je to native query